cardano-cli/src/Cardano/CLI/EraBased/Governance/Actions/Run.hs

```diff
@@ -5,6 +5,9 @@
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeFamilies #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+{-# HLINT ignore "Redundant id" #-}
```

Comment:
is this needed for this PR? it doesn't look like this change uses it

Reply:
Removed.  Thanks!